src/java_bytecode/java_bytecode_parse_tree.h

```diff
@@ -173,8 +173,33 @@ class java_bytecode_parse_treet
     bool is_abstract=false;
     bool is_enum=false;
     bool is_public=false, is_protected=false, is_private=false;
+    bool attribute_bootstrapmethods_read = false;
     size_t enum_elements=0;
 
+    enum class method_handle_typet
+    {
+      BOOTSTRAP_METHOD_HANDLE,
```

Comment:
Given the refactoring seperating out reading arbitrary invoke_dynamic method handles and the specific ones we pass, I think only `LAMBDA_METHOD_HANDLE` and `UNKNOWN_METHOD_HANDLE` are now used?